examples/client/Locomotion/src/Components/Marker/index.js

```diff
@@ -28,11 +28,21 @@ export default ({
     : i18n.t('general.now');
   const typeDetails = {
     [STOP_POINT_TYPES.STOP_POINT_PICKUP]: {
-      icon: pickupIcon,
+      Icon: <SvgIcon
+        Svg={pickupIcon}
+        width={20}
+        height={20}
+        style={{ top: Platform.OS === 'ios' ? -35 : 0 }}
```

Comment:
why -35? because of the notch?

Reply:
I don't know to be honest, I just copied the styles